examples/speaker_tasks/diarization/conf/offline_diarization.yaml

```diff
@@ -31,17 +31,18 @@ diarizer:
   speaker_embeddings:
     model_path: ??? #.nemo local model path or pretrained model name (ecapa_tdnn or speakerverification_speakernet)
     parameters:
-      window_length_in_sec: 1.5 # window length in sec for speaker embedding extraction
-      shift_length_in_sec: 0.75 # shift length in sec for speaker embedding extraction
+      window_length_in_sec: 1.5 # window length(s) in sec for speaker embedding extraction. single floating-point number or a list. Ex) 1.5 or [1.5,1.0,0.5]
```

Comment:
How about make `window_length_in_sec` and `shift_length_in_sec` and `multiscale_weights` clearer? Say single floating number for single scale or a list for multi scale.

Reply:
This will be updated with very concise examples.